CHANGELOG.md

```diff
@@ -5,6 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+# [4.13.2] - 2022-08-24
```

Comment:
At the bottom of the file, need a line like but with the new version

`[4.13.1]: https://github.com/WeInfuse/change_health/compare/v4.13.0...v4.13.1`